### PR TITLE
	Commited changes to add file _targets.R

### PR DESCRIPTION
**Pull request by jholbrook64**
added file _Targets.R to the branch makefile. _Targets.R is used to automate the script my_happy_script.R

### CODE REVIEW

**Comment by jholbrook64** [comment]
Hi! I reached a difficult issue for me to resolve, the function signatures for the function "make_plot" in the _targets.R and code.R scripts seem to match to me, however I get an error returned when trying to build the target that uses that function. The errors depend on how I modify the function signatures but they are either "unused parameter" of eval_data or that eval_data can't be found. (where eval_data is the dataframe being put into the functions) I feel like I am missing an important concept for how the targets read the functions in an R script. Any feedback is appreciated! Below are my error messages: 

Using targets::tar_make()
tar_make()


Using targets Debugger:
tar_make()Error

File: _targets.R [hunk 1]
```diff
@@ -0,0 +1,39 @@
+library(targets)
+files <- c("1_fetch/src/code_to_fetch.R", "2_process/src/code_to_process.R")
+for (file in files) {
```
Comment (jholbrook64) [inline _targets.R:3]: Added separate fetch and process scripts in their respective src directories

File: _targets.R [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    format = "file"
+  ),
+  # Save the model diagnostics
+  tar_target(
+    model_diagnostic_text_txt,
+    generate_model_diagnostics(out_filepath = "2_process/src/", data = eval_data), 
+    format = "file"
+  )
+)
+
```
Comment (jholbrook64) [inline _targets.R:39]: using targets::tar_make(), the code compiles completely:

workingTargetsOutput

File: _targets.R [hunk 3]
```diff
@@ -0,0 +1,36 @@
+library(targets)
+source("code.R")
+tar_option_set(packages = c("tidyverse", "stringr", "sbtools", "whisker"))
+
+list(
+  # Get the data from ScienceBase
+  tar_target(
+    model_RMSEs_csv,
```
[the remark this answers is not in the document]
Comment (jholbrook64) [reply _targets.R:11]: fixed in previous commit

File: _targets.R [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    download_data(out_filepath = "model_RMSEs.csv"),
+    format = "file"
+  ), 
+  # Prepare the data for plotting
+  tar_target(
+    eval_data,
+    process_data(in_filepath = model_RMSEs_csv),
+  ),
+  # Create a plot
```
[the remark this answers is not in the document]
Comment (jholbrook64) [reply _targets.R:20]: fixed in previous commit

**Review by limnoliver — APPROVED** [review]
Nice work! Isn't it fun when it works?!

**Pull request merged**